connectorx-python/connectorx/tests/test_postgres.py

```diff
@@ -1058,3 +1060,84 @@ def test_postgres_partitioned_pre_execution_queries(postgres_url: str) -> None:
         },
     ).sort_values(by=['name']).reset_index(drop=True)
     assert_frame_equal(df, expected, check_names=True)
+
+def test_postgres_inet_type(postgres_url: str) -> None:
```

Comment:
Since we are not following pandas's behaviour by returning the string types, probably it will be better to specify this in the [documentation](https://sfu-db.github.io/connector-x/databases/postgres.html#postgres-pandas-type-mapping), which can be updated by editing the markdown file [here](https://github.com/sfu-db/connector-x/blob/main/docs/databases/postgres.md).